test/integration/IntegrationTestGasComparisons.sol

```diff
@@ -70,49 +71,54 @@ contract IntegrationTestGasComparisons is IntegrationTestHelper {
     }
 
     function testFuzz_wells_WethDaiUsdc_Swap(uint amountIn) public {
-        
         vm.pauseGasMetering();
         uint amountIn = bound(amountIn, 1e18, 1000e18);
 
-        // approve WETH to send to pipeline
-        WETH.approve(address(pipeline), type(uint).max);
-        
+        WETH.approve(address(depot), type(uint).max);
+        deal(address(WETH), address(depot), amountIn * 5);
```

Comment:
remove

Reply:
can remove the deal, but we need to keep the approve as the first call of the `farm` call is transferring the tokens from `msg.sender` to `pipeline` via `depot`.